python/ray/experimental/data/impl/sort.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Any, Callable, TypeVar, Tuple, Union
+
+import numpy as np
+import ray
+from ray.experimental.data.block import Block
+from ray.experimental.data.impl.block_builder import SimpleBlock
+from ray.experimental.data.impl.block_list import BlockList
+
+T = TypeVar("T")
+
+SortKeyT = Union[None, str, List[str], Callable[[T], Any]]
+
+
+def sample_boundaries(blocks: BlockList[T], num_reducers: int) -> List[T]:
+    n_samples = int(num_reducers * 10 / len(blocks))
```

Comment:
How is this number chosen?

Reply:
No good reason; also not sure what's a good heuristic.